exercises/concept/lasagna/src/test/java/utils/ReflectionProxy.java

```diff
@@ -0,0 +1,477 @@
+package utils;
+
+import java.lang.reflect.*;
+
+import static java.lang.Class.forName;
+
+public abstract class ReflectionProxy {
+
+    /**
+     * An instance of the target class (if found)
+     */
+    private Object target;
+
```

Comment:
`target` can be final for this implementation.

Reply:
done